packages/core/upload/server/services/image-manipulation.js

```diff
@@ -174,15 +174,6 @@ const breakpointSmallerThan = (breakpoint, { width, height }) => {
   return breakpoint < width || breakpoint < height;
 };
 
-// TODO V5: remove isSupportedImage
-const isSupportedImage = (...args) => {
-  process.emitWarning(
-    '[deprecated] In future versions, `isSupportedImage` will be removed. Replace it with `isImage` or `isOptimizableImage` instead.'
```

Comment:
Wouldn't that be a great opportunity to write a codemod for this? Seems reasonable since it's related to one of our API

Reply:
Yeah, I'll add one